contracts/utils/deploy.js

```diff
@@ -236,6 +236,56 @@ const sendProposal = async (proposalArgs, description) => {
   log("Done");
 };
 
+async function deployment(deployName, fn) {
```

Comment:
Maybe also add an optional dependencies (list of strings) argument.
And have something like
```
if (dependencies && dependencies.length > 0) {
  main.dependencies = dependencies.
}
```

Reply:
nit: add jsdoc for the new 'deployment' method

Reply:
Thanks, added.